chia/util/initial-config.yaml

```diff
@@ -408,7 +408,7 @@ full_node:
   # How long to wait for a peer connection
   peer_connect_timeout: 30
   # Accept peers until this number of connections
-  target_peer_count: 80
+  target_peer_count: 40
```

Comment:
This should be changed automatically when updating, most users won't look at this